CHANGES.rst

```diff
@@ -9,6 +9,12 @@ detailed information about all changes.
 
 Below is a summary:
 
+Version 0.3.1
+-------------
+* Removed PAYPAL_RECEIVER_EMAIL from settings to allow multiple receiver emails
+  in a single app. Now validation of email must be done once received a
+  'valid tranasaction' signal.
```

Comment:
Typo -  'tranasaction' should be 'transaction', but actually this should be 'valid_ipn_received'.

In addition, we should document the fact that the 'pdt' view is deprecated, and that you must pass the receiver email in the `initial` parameter to `PayPalPaymentsForm`.